packages/mds-utils/package.json

```diff
@@ -22,6 +22,6 @@
     "build": "tsc --build tsconfig.build.json",
     "test": "yarn test:eslint && yarn test:unit",
     "test:eslint": "eslint --fix --ignore-path ../../.gitignore '**/*.ts'",
-    "test:unit": "DOTENV_CONFIG_PATH=../../.env nyc --check-coverage --exclude tests --extension .ts --lines 23 --reporter=text --reporter=html ts-mocha --project ../../tsconfig.json --require tsconfig-paths/register --require source-map-support/register --require dotenv/config --recursive --timeout 5000 tests/**/*.ts"
+    "test:unit": "DOTENV_CONFIG_PATH=../../.env nyc --lines 23 ts-mocha --project ../../tsconfig.json tests/**/*.ts"
```

Comment:
Unrelated, but just noticing a few of these `--lines` thresholds seem a little low

Reply:
Indeed. I made a pass to increase those thresholds that were low unnecessarily. Some packages need more tests. Perhaps tickets should be created to improve coverage on packages below a certain coverage percentage.